src/equations/svaerd_kalisch_1d.jl

```diff
@@ -94,7 +94,7 @@ function initial_condition_manufactured(x, t,
                                         mesh)
     eta = exp(t) * cospi(2 * (x - 2 * t))
     v = exp(t / 2) * sinpi(2 * (x - t / 2))
-    D = 3.0
+    D = 5.0 + 2.0 * cospi(2 * x)
```

Comment:
```suggestion
    D = 5 + 2 * cospi(2 * x)
```
The constants will be promoted automatically but this would also allow running everything with `Float32`